docs/src/pages/[platform]/connected-components/authenticator/customization/overrides/username.react.mdx

```diff
@@ -12,13 +13,15 @@ export default function AuthenticatorWithEmail() {
       // custom username
       username = username.toLowerCase();
       attributes.email = attributes.email.toLowerCase();
```

Comment:
Can you add amend this to `options.userAttributes.email = options.userAttributes.email.toLowerCase();` please?